airflow/dags/ingestion/ask-astro-forum-load.py

```diff
@@ -26,7 +26,7 @@ def get_astro_forum_content():
 
 
 @dag(
-    schedule_interval=schedule_interval,
+    schedule=schedule_interval,
```

Comment:
I agree this change is needed, but I'd like to see what others think about scoping it in this PR rather than a smaller one specifically for this change.

Reply:
Actually I agree. Perhaps let me just revert these and change them later in another PR then. I think when this was brought up before I thought I should change everything but this makes sense as it's bloating this PR review.